modules/oadp-ceph-split.adoc

```diff
@@ -0,0 +1,165 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc
+
+:_content-type: PROCEDURE
+[id="oadp-ceph-split_{context}"]
+= Backing up and restoring data using OADP 1.2 Data Mover and split volumes (CephFS and Ceph RBD)
+
+You can use OpenShift API for Data Protection (OADP) 1.2 Data Mover to back up and restore data in an environment that has _split volumes_, that is, an environment that uses both CephFS and CephRBD.
+
+// [IMPORTANT]
+// ====
+// The CephFS shallow copy feature can only be used for Data Mover backup operations.  The shallow copy volume options are not supported for restore.
+// ====
+
+.Prerequisites
+
+* Ensure a stateful application is running in a separate namespace with PVCs provisioned by both CephFS and CephRBD.
+
+* CephFS is the default `StorageClass` and `VolumeSnapshotClass`.
```

Comment:
Should these prereqs match the similar ones above?

Reply:
I would not assume so -- this is a case where both CephFS and CephRBD are in the same cluster. I did add the `secret` since it's for all DPAs.